### ISSUE TITLE
syntax highliting

### ISSUE THREAD

**Issue by manuelep** [alberthier/git-webui#9]
Hi!
I've just installed git-webui and followed the first basic instruction, it looks great but the first thing I notice is that it doesn't highlight code (at least python) syntax... isn't code syntax highlighting supported or am I missing something?
Thanks a lot.
Best regards

    Manuele

**Comment by alberthier** [comment]
No, you are right : syntax highlighting isn't currently supported.
Would be a great improvement for sure
